### PR TITLE
Extract benchmark utils to separate package

### LINKED ISSUE #361: Create Benchmark package
Might consider pulling out the benchmark framework into a separate package (in lodestar) and holding off on the bls & ssz benchmark for a few days.

### PR DESCRIPTION
**Pull request by mpetrunic**
- extract benchmark utils to separate package
- add setup & teardown methods
- completely abstracted underlining package so we can swap easily

resolves #361

### CODE REVIEW

File: packages/benchmark-utils/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  // Create the current benchmark folder
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir);
+  }
+  return dir;
+};
+
+export const writeReport = (file: string, data: string) => {
+  console.log(data);
```
Comment (GregTheGreek) [inline packages/benchmark-utils/src/index.ts:29]: remove

**Review by GregTheGreek — CHANGES_REQUESTED** [review]
LGTM 🚀 Just a console log

File: packages/benchmark-utils/src/index.ts (hunk 1 above)
Comment (mpetrunic) [reply packages/benchmark-utils/src/index.ts:29]: But I actually wan't to see result in console as well :sweat_smile:

Comment (GregTheGreek) [reply packages/benchmark-utils/src/index.ts:29]: Use a logger? Lol idk what's the output

Comment (mpetrunic) [reply packages/benchmark-utils/src/index.ts:29]: same thing that goes into file. We dont have logger available there nor there is a point in logger for dev tool haha

Comment (GregTheGreek) [reply packages/benchmark-utils/src/index.ts:29]: oh ya true..

**Review by GregTheGreek — APPROVED** [review]

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**